packages/functional_widget/README.md

```diff
@@ -217,6 +217,20 @@ There are a few ways to do so:
   Widget example(int foo, String bar) => Container();
   ```
 
+- With parameters on the `@FunctionalWidget` decorator that will export the generated widget for private functions:
```

Comment:
Coming back to this, I don't think this is the correct place to mention such information.

The context here is about changing the widget type, not its privacy/name